compiler/qsc_packages/src/lib.rs

```diff
@@ -157,8 +163,556 @@ mod tests {
+                                        []: [
+                                            Open {
+                                                namespace: NamespaceId(
+                                                    24,
+                                                ),
+                                                span: Span {
+                                                    lo: 0,
+                                                    hi: 40,
+                                                },
+                                            },
+                                        ],
+                                    },
+                                    tys: {},
+                                    terms: {},
+                                    vars: {},
+                                    ty_vars: {},
+                                },
+                                Scope {
+                                    span: Span {
+                                        lo: 14,
+                                        hi: 40,
+                                    },
+                                    kind: Callable,
+                                    opens: {},
+                                    tys: {},
+                                    terms: {},
+                                    vars: {},
+                                    ty_vars: {},
+                                },
+                                Scope {
+                                    span: Span {
+                                        lo: 38,
+                                        hi: 40,
+                                    },
+                                    kind: Block,
+                                    opens: {},
+                                    tys: {},
+                                    terms: {},
+                                    vars: {},
+                                    ty_vars: {},
+                                },
+                            ],
+                        },
+                    },
+                    assigner: Assigner {
+                        next_node: NodeId(
+                            9,
+                        ),
+                        next_item: LocalItemId(
+                            2,
+                        ),
+                    },
+                    sources: SourceMap {
+                        sources: [
+                            Source {
+                                name: "test",
+                                contents: "@EntryPoint() operation Main() : Unit {}",
+                                offset: 0,
+                            },
+                        ],
+                        common_prefix: None,
+                        entry: None,
+                    },
+                    errors: [],
+                    dropped_names: [],
+                },
+                [],
+            )
+        "#]].assert_debug_eq(&compiled);
```

Comment:
Yeah, I don't like this currently. My changes caused this expect string to be updated, and I changed the formatting because I thought diffing would be easier (it wasn't). You can change it back, but I like your suggestion even better to simplify it.